### ISSUE TITLE
Semantic Exploit Naming

### ISSUE THREAD

**Issue by HenryHoggard** [ncrocfer/csr2f#1]
Have you considering using meaningful file names for exploits rather than numbers?

eg 

wordpress_download_plugin_1.4.5.csrf

rather than 
99.csrf

It would be a lot easier to manage and search for things.

**Comment by ncrocfer** [comment]
Yes it's a good idea. I already thought, the numbers are not very practical and users can easily make a mistake by typing the exploit.

I think it will be a good feature with an autocomplete system.
